packages/nuxt/src/runtime/utils.ts

```diff
@@ -26,3 +30,26 @@ export function extractErrorContext(errorContext: CapturedErrorContext): Context
 
   return dropUndefinedKeys(structuredContext);
 }
+
+/**
+ * Adds Sentry tracing <meta> tags to the returned html page.
+ *
+ * Exported only for testing
+ */
+export function addSentryTracingMetaTags(head: NuxtRenderHTMLContext['head']): void {
+  const activeSpan = getActiveSpan();
+  const rootSpan = activeSpan ? getRootSpan(activeSpan) : undefined;
+
+  if (rootSpan) {
+    const traceParentData = spanToTraceHeader(rootSpan);
+    const dynamicSamplingContext = dynamicSamplingContextToSentryBaggageHeader(
+      getDynamicSamplingContextFromSpan(rootSpan),
+    );
+    const contentMeta = [
+      `<meta name="sentry-trace" content="${traceParentData}"/>`,
+      `<meta name="baggage" content="${dynamicSamplingContext}"/>`,
+    ];
+
+    head.push(...contentMeta);
```

Comment:
super-l: I think we can skip this array, we can just `head.push('<meta...')` twice, safe a tiny bit of processing? 😅